client.go

```diff
@@ -283,6 +296,51 @@ func (c *Client) DownloadState(organization, workspace, stateVersion string) ([]
 	return raw, err
 }
 
+// DownloadLatestState downloads the raw state file from Terraform Enterprise
+// Requires 3 requests:
+// - GetLatestStateVersion (2)
+// - download from HostedStateDownloadURL
+func (c *Client) DownloadLatestState(organization, workspace string) ([]byte, error) {
+	sv, err := c.GetLatestStateVersion(organization, workspace)
+	if err != nil {
+		return nil, err
+	}
+
+	var resp *http.Response
```

Comment:
how about we put this in it's own private function since we're doing it twice in different functions (downloadlateststate and downloadstate). could be something like: 

b, err := downloadStateVersion(sv)